includes/Modules/Analytics/Account_Ticket.php

```diff
@@ -157,6 +165,28 @@ public function set_timezone( $timezone ) {
 		$this->timezone = (string) $timezone;
 	}
 
+	/**
+	 * Gets the enabled state of enhanced measurement for the data stream.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @return bool $enabled Enabled state.
+	 */
+	public function get_enhanced_measurement_stream_enabled() {
+		return $this->enhanced_measurement_stream_enabled;
+	}
+
+	/**
+	 * Sets the enabled state of enhanced measurement for the data stream.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @param bool $enabled Enabled state.
+	 */
+	public function set_enhanced_measurement_stream_enabled( $enabled ) {
```

Comment:
This is another simplification of the IB which fits into what we're currently doing a bit more cleanly. There are other bits of state in here which are not strictly related to account ticket creation, so we extend that a bit since this is mostly a helper object.